libnopegl/nodes.specs

```diff
@@ -648,16 +648,6 @@
         "desc": "only the luma channel"
       }
     ],
-    "framebuffer_features": [
-      {
-        "name": "depth",
-        "desc": "add depth buffer"
-      },
-      {
-        "name": "stencil",
-        "desc": "add stencil buffer"
-      }
-    ],
```

Comment:
probably need a mention in the changelog

Reply:
Done.